pages/index.js

```diff
@@ -17,14 +39,25 @@ export default function Home() {
 
 const Wrap = styled.div`
   width: 100vw;
-  height: 100vh;
+  min-height: 100vh;
   background: url(/images/bgImage.png);
   backdrop-filter: blur(2px);
 `;
 
 const WrapMask = styled.div`
   width: 100vw;
-  height: 100vh;
+  min-height: 100vh;
   background: rgba(253, 231, 190, 0.4);
   backdrop-filter: blur(40px);
+  padding-bottom: 1.5rem;
+`;
+
+const WelcomeStyle = styled.div`
+  padding-top: 2rem;
+  padding-left: 1.5rem;
+`;
+
+const DateDisplayStyle = styled.div`
+  margin-top: 0.3rem;
+  padding-left: 1.5rem;
```

Comment:
Here you can use the grid tecniques as well so the wrapper assigns the spacing of the child elements.